test/param.c

```diff
@@ -1528,7 +1531,12 @@ ODBC_TEST(odbc45)
   for (i= 0; i<sizeof(XpctdValue); ++i)
   {
     CHECK_STMT_RC(Stmt, SQLFetch(Stmt));
+  #if defined(__s390x__)
+    SQLGetData(Stmt, 1, SQL_C_BIT, &value, sizeof(value), 0);
```

Comment:
It doesn't look, that for test it's important to fetch values as ints, thus it can be just changed to fetch values as SQL_C_BIT